charts/super-agent/charts/super-agent-deployment/templates/configmap-superagent-config.yaml

```diff
@@ -10,4 +10,9 @@ apiVersion: v1
 data:
   local_config: |
     {{- include "newrelic-super-agent.config.content" . | nindent 4 }}
+  # this is intended for dev use only
+  {{- with (.Values.development).dynamicAgentType }}
+  dynamic_agent_type: |
```

Comment:
couldn't we use extra volumes for this?

Reply:
🤔 Yes, I wanted to have something easy for that external team. I'll try with that to see if we can avoid this extra "hidden" feature!